Evo_Game_Function

```diff
@@ -0,0 +1,316 @@
+Evo_Game <- function(nrow = 4, ncol = 4, nFood = 5, nAgent = 2, tick = 2) {
+  
```

Comment:
On a general note: 
If you delete and create a whole new file, it makes it hard for us as reviewers to see what exactly you changed. 
If possible, try to change your code and rename you code in the original file, so that reviewers can follow your progress and train of thoughts.